THEME-SONGS.md

```diff
@@ -0,0 +1,45 @@
+# Theme songs template
+
+In order to properly link to a wrestler's theme songs, use the following template:
+
+```
+## In wrestling
+
+* Entrance music:
+- "Title 1" by Artist 1
+ {{ org_badge(orgs=['ppw']) }} (20??-20???) <br>
+ {{ music(yt='')}}
+ {{ music(spot='')}}
+ {{ music(apple='')}}
+  - "Title 2" by Artist 2
+ {{ org_badge(orgs=['mzw', 'low', 'kpw']) }} (20??-present, in tag team with This Guy) <br>
+ {{ music(yt='')}}
+ {{ music(spot='')}}
+ {{ music(apple='')}}
+  - "Title 3" by Artist 3
+ {{ org_badge(orgs=['ppw']) }} (20??, as Early Gimmick) <br>
+ {{ music(yt='')}}
+ {{ music(apple='')}}
+```
+
+* Place the template in the _In wrestling_ section of the article; in the section is not present, go ahead and add it. It should be placed near the end of the article, after the wrestler's bio but before _Internet presence_ (which in turn comes before _References_).
+
+* Please pay attention to the spaces af the beginning of individual lines. There are two spaces before the line with the title/artist and one space before each line beginning with curly brackets.
+
+* Each badge needs to be separately enclosed in apostrophes. Badges and apostrophes are inside square brackets.
+
+* Regardless of how many badges there are, it's always ```orgs=```, not ```org=```. Removing the _s_ will cause an error.
```

Comment:
For a single one, `org='ppw'` works, but for multiple ones, must be `orgs=['A', 'B', 'C']`